pkg/ui/src/components/input/input.styl

```diff
@@ -61,3 +61,8 @@
     &--error-message
       color $colors--functional-red-3
       position absolute
+  &__password
+    font-family sans-serif
```

Comment:
Is it possible to use font styles from design system? (user can click "show password" and font styles with login input will be different)

Reply:
no, font from design system is the reason of this bug.

Reply:
Then i would suggest to use the same font for "login" field as well just for consistency (In case these fonts visibly different). Other than that, it LGTM!

Reply:
I see what @koorosh is saying, but I think it would be more correct to have the password input component use this style (`font-style sans-serif`) when the input type is `password`, and inherit the font when the type is `text`.

I think the easiest way to do that would be to change this line in [`passwordInput.tsx`](https://github.com/cockroachdb/cockroach/blob/3c8736df144ecadfb717e99b35187c5678f88d47/pkg/ui/src/components/input/passwordInput.tsx#L66),

```tsx
const classes = cn(className, "crl-input", { "crl-input__password": !showPassword });
```